cross/boto/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME = boto
+PKG_SHORT_VERS = 2.38
```

Comment:
You don't seem to need `PKG_SHORT_VERS` anywhere for Boto. Please just refer to `PKG_VERS=2.38.0` if that's correct.